lib/build.js

```diff
@@ -39,8 +39,12 @@ var baseRequireConfig = {
       contents = contents.replace(/\}\);\s*?$/, '');
 
       if (!contents.match(/Modernizr\.add(Async)?Test\(/)) {
-      // remove last return statement and trailing })
-        contents = contents.replace(/return.*[^return]*$/, '');
+        // remove last return statement and trailing })
+        contents = contents.replace(
+          /^.*return.*$/,
+          function(match) {
+            return m.split("return").slice(0, -1).join('return').trim();
+          });
```

Comment:
Another option would be implementing the following replace call:

```javascript
contents = contents.replace(/^(.*)return.*$/, '$1');
```

The only caveat is, that `'return yes return no'` becomes `'return yes '` with a trailing space, not sure if that would be a problem. Using the callback version is definitely the most flexible.